workshop/content/my-first-enclave/cryptographic-attestation/_index.en.md

```diff
@@ -7,6 +7,8 @@ weight = 14
 
 In this section, you will gain experience with a sample sensitive data processing application that makes use of Nitro Enclaves cryptographic attestation and AWS KMS to isolate sensitive data processing within the boundary of your enclave. This sample demonstrates how cryptographic attestation can be used to protect the privacy of sensitive data. Initially, you will build and deploy this sample application without taking advantage of cryptographic attestation in order to understand its operation. In the second phase of this section, you will enable cryptographic attestation and observe how it can be used to enhance the isolation of sensitive data processing workloads in a Nitro Enclave.
 
+![Architecture diagram](/images/cryptographic-attestation-arch.png)
```

Comment:
Would it make sense to put this on line 29 so it is located alongside the text description for the application?

Reply:
Sounds good. I'll move it.